example.py

```diff
@@ -15,6 +16,12 @@
 model_name = "model1"
 os.makedirs(model_dir, exist_ok=True)
 
+# load args from user's command input
+parser = argparse.ArgumentParser()
+parser.description='To customize your map: use command python example.py -reset=True'
+parser.add_argument("-reset", "--input if reset map", help="False- using default map, True - Create your own map", dest="ifreset", type=bool, default=False)
+args = parser.parse_args()
+
```

Comment:
This is great! Is there a way to include it in the actual environment instead of this example file though? People may or may not use this file.

Reply:
Are there other files people can use to test the environment? If they are writing their own code to access the environment they may directly hardcoding this parament(as original way) / added this part of code and input parameters in the terminal.

Reply:
Gotcha. People might be writing their own files to play with the simulation but its okay. Let's leave it in here for now.